### PR TITLE
Added pitfalls

### LINKED ISSUE #219: Pitfalls
As a dwarf, I want there to be pitfalls in the game, so that I have more ways to hinder the player.

This task has the following subtasks:
- #220 Pitfalls - Pitfall Entity
- #221 Pitfalls - Player Interaction

### LINKED ISSUE #220: Pitfalls - Pitfall Entity
Implement the PitfallCover entity. This entity is drawn like a normal tile for the player, but has a different interaction. This task includes updating the web interface to support the pitfall entity.

This is a subtask of #219 - Pitfalls

### LINKED ISSUE #221: Pitfalls - Player Interaction
When walking over a pitfall, the player should fall through it.

This is a subtask of #219 - Pitfalls

### PR DESCRIPTION
**Pull request by ChielBruin**
Added a pitfall entity that drops the player through the maze. 
The player is respawned after 2 seconds. When health is introduced, the player should be damaged.

Closes #219, closes #220 and closes #221

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/model/entities/Pitfall.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		VRPlayer p = Main.getInstance().getCurrentGame().getPlayer();
+		p.move(0, -2f, 0);
+	}
+
+	@Override
+	public Spatial getSpatial() {
+		if (sp != null) return sp;
+		Box b = new Box(.4f, .01f, .4f);
+		sp = new Geometry("plate", b);
+		Material mat = new Material(Main.getInstance().getAssetManager(), "Common/MatDefs/Light/Lighting.j3md");  // create a simple material
```
Comment (Geweldig) [inline src/main/java/nl/tudelft/contextproject/model/entities/Pitfall.java]: Is this comment still needed?

**Comment by Geweldig** [comment]
Can be merged after my issues have been adressed.

**Comment by Geweldig** [comment]
Should be merged asap.

**Comment by Taeir** [comment]
Looks good, this can be merged.

**Pull request merged**